### ISSUE TITLE
Accessibility: MAS26: No Skip link for the page.

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#2]
#A11yMAS
#MAS26
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1.Open  https://marketplace.visualstudio.com/VSCode
2.Check for the skip links.

Note: Same issue is observed in below link
https://marketplace.visualstudio.com/items?itemName=donjayamanne.python

**Actual Result:**
No skip link is defined for the page.

**Expected Result:**
There should be at least one skip  link defined for the page .

**User Impact:**
Visually Impaired users relying on screen readers need to navigate throughout page ,if there is no skip link.
![no skip content for the link](https://cloud.githubusercontent.com/assets/24825441/21672108/00f022ea-d346-11e6-9de6-f010d24e2cad.png)
